_clubs/neighborhood-watch-and-emergency-preparedness.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Neighborhood Watch and Emergency Preparedness
+date: 2019-06-10 07:25
+author: Klahanie
+comments: true
+categories: [clubs]
+link: true
+description: Neighborhood Watch helps us keep our property safe. 
+thumbnail: /images/Neighborhood-Watch.png
+banner: /images/family-playing-outdoors.jpg
+---
+### Neighborhood Watch &amp; Emergency Preparedness 
+
+<blockquote class="blockquote">If You See Something, Say Something</blockquote>
+
+<img src="{{site.url}}images/Neighborhood-Watch.png" class="float-right w-20">
+**Neighborhood Watch (NW) begins when you and your neighbors. gather in another’s home or connect via a summer BBQ/Block Party and**

```

Comment:
"Neighborhood Watch (NW) begins when you and your neighbors gather"
I'm not sure what this is trying to say, but we're missing text at the end.

Reply:
Perhaps the first bullet is meant to go with the explanation above? 
"three-minute"
Missing a word after "serve as your NW/EP"? Maybe "representative"?
"(She/he will..." sentence doesn't make sense with the bulleted list that follows.
Change comma to period at end of second bulleted item.
This entire section could stand a rewrite.